packages/wxt/src/core/utils/building/generate-wxt-dir.ts

```diff
@@ -201,7 +201,6 @@ async function getGlobalsDeclarationEntry(): Promise<WxtDirFileEntry> {
     path: 'types/globals.d.ts',
     text: [
       '// Generated by wxt',
-      'export {}',
```

Comment:
Very interesting, I think I copied this from Nuxt? I had always questioned why they had the export there, but I remember it working at some point lol.